gallery/visualtests/nxRadio.js

```diff
@@ -15,84 +17,94 @@ describe('NxRadio', function() {
     });
 
-    it(`has a thick blue border, white background, with a light blue outer border and glow
-      when clicked and focused`, async function() {
-      const focusSelector = `${selector} input`,
-          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
+    it(`has a thick blue border, white background, with a blue outer border '
+    + 'when clicked and focused`, async function() {
+      const focusSelector = `${simpleExampleLabelSelector} input`,
+          [targetElement, focusElement] = await waitAndGetElements(simpleExampleLabelSelector, focusSelector);
 
       await targetElement.click();
       await moveMouseAway();
       await focusElement.focus();
 
-      await checkScreenshot(targetElement);
+      await checkScreenshotWithOutset(targetElement, OUTSET);
     });
 
-    it(`has a thick blue border and white background with a light blue outer border
+    it(`has a thick blue border and white background with a blue outer border
       when clicked, focused, and hovered`, async function() {
-      const focusSelector = `${selector} input`,
-          [targetElement, focusElement] = await waitAndGetElements(selector, focusSelector);
+      const focusSelector = `${simpleExampleLabelSelector} input`,
+          [targetElement, focusElement] = await waitAndGetElements(simpleExampleLabelSelector, focusSelector);
 
       await targetElement.click();
       await focusElement.focus();
       await targetElement.hover();
 
-      await checkScreenshot(targetElement);
+      await checkScreenshotWithOutset(targetElement, OUTSET);
     });
 
-    it('has a light blue outer border and glow when focused', focusTest(selector));
-    it('has a light blue outer border and a dark border when focused and hovered', focusAndHoverTest(selector));
+    it('has a blue outer border when focused', focusTest(simpleExampleLabelSelector, undefined, OUTSET));
+    it('has a blue outer border and a dark border when focused and hovered',
+        focusAndHoverTest(simpleExampleLabelSelector, undefined, OUTSET)
+    );
+  });
+
+  describe('Attribute-Disabled NxRadio and Attribute-Disabled-Checked', function() {
+    it('looks disabled by default', simpleTest(disabledExampleLabelSelector, OUTSET));
   });
 
   describe('Attribute-Disabled NxRadio', function() {
-    it('looks disabled by default', simpleTest(disabledSelector));
-    it('looks disabled when hovered', hoverTest(disabledSelector));
+    it(
+        'looks disabled when hovered',
+        hoverTest(disabledExampleLabelSelector, undefined, undefined, OUTSET)
+    );
   });
 
   describe('Attribute-Disabled-Checked NxRadio', function() {
-    it('looks disabled by default', simpleTest(disabledCheckedSelector));
-    it('looks disabled when hovered', hoverTest(disabledCheckedSelector));
+    it(
+        'looks disabled when hovered',
+        hoverTest(disabledExampleLabelCheckedSelector, undefined, undefined, OUTSET)
+    );
```

Comment:
Same kind of formatting issue here.

Reply:
> Same kind of formatting issue here.

Fixed here thanks!: https://github.com/sonatype/sonatype-react-shared-components/pull/986/commits/df89b5ac89996ba8d249763625489ee14c687b49